src/components/ChannelPage.vue

```diff
@@ -27,6 +27,13 @@
                     @click="subscribeHandler"
                 ></button>
 
+                <button
+                    v-if="subscribed"
+                    v-t="'actions.add_to_group'"
```

Comment:
Should this be manage group instead, considering you could technically use this to remove it from a channel group too?

Reply:
Probably something like "Change group membership", but I think  "Add to group" is just more self-explaining (and shorter) in that case.

"Manage group" sounds wrong because you don't manage a group, you change whether the channel is part of the groups.